src/OSPSuite.UI/UIConstants.cs

```diff
@@ -39,24 +39,23 @@ public static class Size
          public static readonly int EMPTY_SPACE_HEIGHT = ScaleForScreenDPI(20);
          public static readonly int RADIO_GROUP_HEIGHT = ScaleForScreenDPI(24);
          public static readonly int LARGE_BUTTON_WIDTH = ScaleForScreenDPI(150);
-         //DO NOT SCALE HEIGHT WITH DPI of BUTTONS
-         public static readonly int LARGE_BUTTON_HEIGHT = 29;
+         public static readonly int LARGE_BUTTON_HEIGHT = ScaleForScreenDPI(29);
```

Comment:
@Yuri05 I am now scaling everything with a DPI and I added another scale factor for very large DPI ratio